client/app/bundles/course/group/forms/GroupCreationForm.jsx

```diff
@@ -148,65 +141,119 @@ const GroupCreationForm = ({
     }
   }, [numToCreate, conflictingNames]);
```

Comment:
Add `dispatch` (turns out I missed it out myself when I first coded this haha)

Hmm I can't remember why I omitted `isSingle`, but I think it should be safe to add it into the dependencies also.

Reply:
Added!